### ISSUE TITLE
Dropdown - VoiceOver doesn't read a list of drop-down items while navigating through items in a drop-down list

### ISSUE THREAD

**Issue by JaneSjs** [surveyjs/survey-library#7882]
The issue is reported at T16900 - Accessibility issues
https://surveyjs.answerdesk.io/internal/ticket/details/T16900.

However, note that NVDA reads [drop-down items](https://surveyjs.io/form-library/examples/create-dropdown-menu-in-javascript/reactjs). Perhaps, the issue is specific to VoiceOver.

![image](https://github.com/surveyjs/survey-library/assets/22372972/231a9d93-5202-44d7-b252-9f66838917f4)

**Comment by dmitry-kurmanov** [comment]
https://github.com/tailwindlabs/headlessui/issues/2129

**Comment by JaneSjs** [comment]
Upon the research, we determined that the VoiceOver cannot read drop-down items because of the issue reported at [# Accessibility issue with MacOS Voiceover support for editable Combobox with grid popup in safari](https://forums.developer.apple.com/forums/thread/706788). Unfortunately, we cannot address this issue at the level of our drop-down component.
